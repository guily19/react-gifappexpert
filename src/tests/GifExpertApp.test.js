import { shallow } from 'enzyme';
import { GifExpertApp } from '../GifExpertApp';

describe('<GifExpertApp />', () => {

  test('<GifExpertApp /> renders', () => {
    const wrapper = shallow(<GifExpertApp />);
    expect( wrapper ).toMatchSnapshot();
  });

  test('<GifExpertApp /> renders with defaultCategories', () => {
    const wrapper = shallow(<GifExpertApp defaultCategories={['Cat1','Cat2']}/>);
    expect( wrapper ).toMatchSnapshot();
    expect( wrapper.find('GifGrid').length).toBe(2);
  });
});