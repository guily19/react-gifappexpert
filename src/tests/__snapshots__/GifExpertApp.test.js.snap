// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GifExpertApp /> <GifExpertApp /> renders 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <hr />
  <AddCategory
    setCategories={[Function]}
  />
  <ol />
</Fragment>
`;

exports[`<GifExpertApp /> <GifExpertApp /> renders with defaultCategories 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <hr />
  <AddCategory
    setCategories={[Function]}
  />
  <ol>
    <GifGrid
      category="Cat1"
      key="Cat1"
    />
    <GifGrid
      category="Cat2"
      key="Cat2"
    />
  </ol>
</Fragment>
`;
